socket.go

```diff
@@ -176,11 +176,13 @@ func NewVirtioSocketListener(handler func(conn *VirtioSocketConnection, err erro
 
 //export shouldAcceptNewConnectionHandler
 func shouldAcceptNewConnectionHandler(listenerPtr, connPtr, devicePtr unsafe.Pointer) C.BOOL {
-	_ = devicePtr // NOTO(codehex): Is this really required? How to use?
```

Comment:
please revert this?

Reply:
I removed it from my PR, though I'm not sure what this is needed for? To avoid some linting error about unused variable ?